chains/solana/contracts/programs/ccip-offramp/src/instructions/v1/execute.rs

```diff
@@ -31,9 +30,21 @@ impl Execute for Impl {
         report_context_byte_words: [[u8; 32]; 2],
         token_indexes: &[u8],
     ) -> Result<()> {
-        let execution_report =
-            ExecutionReportSingleChain::deserialize(&mut raw_execution_report.as_ref())
-                .map_err(|_| CcipOfframpError::FailedToDeserializeReport)?;
```

Comment:
Qq, since I see this error mapping was removed, what is the new error that is thrown if the report input is bad and fails to deserialize correctly?

Reply:
Ah my bad, I forgot to replace this. It will bubble up some Borsh error, so I'll fix it to be as it used to.